webknossos/webknossos/dataset/view.py

```diff
@@ -439,7 +445,11 @@ def for_zipped_chunks(
             job_args.append((source_chunk_view, target_chunk_view, i))
 
         # execute the work for each pair of chunks
-        wait_and_ensure_success(executor.map_to_futures(work_on_chunk, job_args))
+        if executor is None:
+            for args in job_args:
+                work_on_chunk(args)
```

Comment:
I added this as a simple for-loop (as stated in the issue). An alternative would be to use `get_executor_for_args(None)` per default.